app/services/idv/financials_step.rb

```diff
@@ -1,32 +1,44 @@
 module Idv
   class FinancialsStep < Step
-    def complete?
-      idv_form.finance_type && idv_session.financials_confirmation.try(:success?) ? true : false
+    def submit
+      if complete?
+        @success = true
+        idv_session.financials_confirmation = true
```

Comment:
Note that switching to a `Bool` here instead of the `Proofing::Confirmation` object means that the complete vendor response is lost from the `idv_session`. I had been keeping it around in `idv_session` until we figured out exactly how much we needed to log from it via analytics in order to track failure/success rationales, possibly to display back via UI to user. If we do want to drop the `Confirmation` object entirely, then we'll need to make sure we instead do all the mining in the vendor gem and expose it some other way to analytics.